README.md

```diff
@@ -181,6 +264,31 @@ The following methods on `String.prototype` will be modified to check for [[RegE
 
 `RegExp[`@@species`]` will be removed.
 
+
+### Example
+
+Before this change:
+
+```js
+class R extends RegExp {
+  [Symbol.match](str) {
+    return "overridden";
+  }
+}
+console.log("some string".match(new R("foo")))     // "overridden"
+```
+
+After this change:
+
+```js
+class R extends RegExp {
+  [Symbol.match](str) {
+    return "overridden";
+  }
+}
+console.log("some string".match(new R("foo")))     // null
+```
```

Comment:
This example doesn't capture the intent. Sorry about the confusion for this section -- I don't think we should remove `@@match` per #2 and #4, but override checks for `exec` and `flags`, etc.